pages/checkout.vue

```diff
@@ -327,13 +316,20 @@ export default {
     ]
 
     const personalDetails = ref({ firstName: "", lastName: "", email: "" })
+    let cardDetails = ref({
```

Comment:
this should be a generic variable, "paymentDetails" = ref({})

Reply:
Done